packages/devtools/lib/src/logger.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Handles console level logging for DevTools.
+class Logger {
+  static void log(String message, {LogLevel level = LogLevel.debug}) {
+    switch (level) {
+      case LogLevel.debug:
```

Comment:
use config specific imports here to forward to 
window.console.log, window.console.warn
etc in the browser.

Reply:
removed logger code from this PR. Will add in a follow up.